src/app-logic/url-handling.js

```diff
@@ -328,6 +329,7 @@ export function getQueryStringFromUrlState(urlState: UrlState): string {
         urlState.profileSpecific.full.localTrackOrderByPid,
         urlState.profileSpecific.full.localTrackOrderChangedPids
       );
+      baseQuery.tabID = urlState.profileSpecific.full.tabFilter || undefined;
```

Comment:
I'd do:
```suggestion
      baseQuery.tabID = urlState.profileSpecific.full.tabFilter ?? undefined;
```
because tabFilter is a number and could be theorically 0

Reply:
Good catch, changed it. Thanks!